src/components/navigation/navigation-marker/navigation-marker.ts

```diff
@@ -68,62 +59,37 @@ export class SbbNavigationMarkerElement extends SlotChildObserver(LitElement) {
     this.reset();
     action.active = true;
     this._currentActiveAction = action;
-    this._hasActiveAction = true;
+    setTimeout(() => this._setMarkerPosition());
```

Comment:
should we use updatescheduler here? at least it did no cause any problems so far.